main/actions/src/main/scala/sbt/Compiler.scala

```diff
@@ -80,6 +95,13 @@ object Compiler {
       val scalac = scalaCompiler(instance, cpOptions)
       new Compilers(scalac, javac)
     }
+  @deprecated("Deprecated in favor of new sbt.compiler.javac package.", "0.13.8")
+  def compilers(instance: ScalaInstance, cpOptions: ClasspathOptions, javac: JavaTool, ivyConfiguration: IvyConfiguration)(implicit app: AppConfiguration, log: Logger): Compilers =
```

Comment:
It is used by (line 72 in this file)
```scala
def compilers(instance: ScalaInstance, cpOptions: ClasspathOptions, javaHome: Option[File], ivyConfiguration: IvyConfiguration)(implicit app: AppConfiguration, log: Logger): Compilers
```
What is supposed to be the replacement for this method? `sbt.compiler.MixedAnalyzingCompiler`?